packages/hocs/withFocusTrap.jsx

```diff
@@ -1,27 +1,47 @@
 /* eslint-disable jsx-a11y/no-noninteractive-tabindex */
-import React, { useRef } from 'react'
+import React, { useRef, useEffect } from 'react'
+import PropTypes from 'prop-types'
 
 // Returns an array of focusable elements in the order they are found in c
-const getFocusable = c =>
-  c.querySelectorAll('button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"])')
+const selector =
+  'button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"]), audio[controls], video[controls], [contenteditable]:not([contenteditable=false]'
+const getFocusable = c => c.querySelectorAll(selector)
 
 const withFocusTrap = Component => {
   const WithFocusTrap = props => {
     const componentRef = useRef(null)
 
+    useEffect(() => {
+      // setting the focus to the first focusable element
+      const focusableElements = getFocusable(componentRef.current)[0]
+      if (focusableElements) focusableElements.focus()
+    })
```

Comment:
This should only run on mount. Please add `[]` as the second parameter to `useEffect`